.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ env:
   # Otherwise, set variable to the commit of your branch on
   # opentelemetry-python-contrib which is compatible with these Core repo
   # changes.
-  CONTRIB_REPO_SHA: bcec49cf2eccf8da66c9e63b9836ea8a20516efc
+  CONTRIB_REPO_SHA: 61ae201677be80e333746f06bc9485fa8856f42c
```

Comment:
Updated to show that tests pass, will revert once approved.

Reply:
As soon as your Contrib repo PR gets merged, you can't revert this. It either has to be `master` or this commit. I think leaving it as this commit is fine so that you don't have to change anything!

Reply:
I thought the flow was supposed to be the reverse i.e, core repo gets merged first and then contrib.